app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str, health: int = 100, hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+        # check Let’s check that the creature hasn’t died.
+        self.update_status()
```

Comment:
There is no need to place comments and update the status upon creating new animals since it is not mentioned in the task description. Please, remove redundant code